experimenter/experimenter/experiments/tests/test_models.py

```diff
@@ -3188,8 +3188,8 @@ def test_nimbus_branch_screenshot_upload_to(self, mock_uuid4):
     def test_nimbus_branch_screenshot_delete_previous_on_save_change(self, mock_uuid4):
         with mock.patch.object(self.screenshot.image.storage, "delete") as mock_delete:
             mock_uuid4.return_value = "predictable"
-            expected_filename = os.path.join(
-                self.experiment.slug, f"{mock_uuid4.return_value}.png"
+            expected_filename = str(
+                Path(self.experiment.slug) / f"{mock_uuid4.return_value}.png",
```

Comment:
Yeah I was thinking about this too about when it's appropriate to put everything inside the constructor vs using the slashes.  But looking at it here I think it might be the case that if you have multiple elements on teh line you're constructing, you can just put them all in the constructor, and if you're later just referring to something that's already a `Path` then it's appropriate to use the slashes.